### PR TITLE
fix: update info sheet and empty sites sheet to match new designs

### LINKED ISSUE #490: design update: home screen
we need to implement the design changes on the home screen:
- content of the info bottom sheet and empty sites list bottom sheet has changed
- sites list bottom sheet initial snap point should be set to 50%
- sites list bottom sheet no longer has a create site button
- sites list bottom sheet title is now "Sites List"

### PR DESCRIPTION
**Pull request by paulschreiber**
## Description
Update info sheet and empty sites sheet to match new designs

### Related Issues
Fixes #490.

### CODE REVIEW

File: dev-client/src/screens/HomeScreen.tsx [hunk 1]
```diff
@@ -275,50 +280,67 @@ const LandPKSInfo = () => {
...
-          />
+              icon: <LinkNewWindowIcon />,
+            }}>
+            <Text bold>first</Text>
+            <Text
+              underline
+              onPress={() => Linking.openURL(t('home.info.link_url'))}
+              color="primary.main">
+              link_text
+            </Text>
```
Comment (david-code) [inline dev-client/src/screens/HomeScreen.tsx:342]: question: Why not use the `Link` component here?

File: dev-client/src/screens/HomeScreen.tsx [hunk 2]
```diff
@@ -275,50 +280,67 @@ const LandPKSInfo = () => {
...
+          </Text>
+        </HStack>
+        <HStack key={3}>
+          <Text variant="body1" mr={2}>
+            3
+          </Text>
+          <Text variant="body1" mr={2}>
+            {t('home.info.list3')}
+          </Text>
+        </HStack>
```
Comment (david-code) [inline dev-client/src/screens/HomeScreen.tsx:329]: suggestion:
```suggestion
    <FlatList 
        data={["home.info.list1", "home.info.list2", "home.info.list2"]} 
        renderItem={({index, item}) => (
        <HStack>
            <Text variant="body1" mr={2}>{index}</Text>
            <Text variant="body1" mr={2}>{t(item)}</Text>
        </HStack>)} 
        keyExtractor={(item) => item} 
    />
```

(I didn't test it out, so there may be some syntax issues).

File: dev-client/src/components/home/BottomSheet.tsx [hunk 3]
```diff
@@ -53,12 +56,18 @@ const EmptySiteMessage = () => {
...
-      <Text>{'\n\n'}</Text>
-      <Link isUnderlined={false} _text={{variant: 'body1'}}>
-        {t('site.empty.link') + ' '}
-        <Icon name="open-in-new" />
-      </Link>
+      <Trans
+        i18nKey="site.empty.info"
+        components={{
+          icon: <LinkNewWindowIcon />,
+        }}>
```
Comment (david-code) [inline dev-client/src/components/home/BottomSheet.tsx:63]: comment: Nice, I wasn't sure how to use the `Trans` component.

File: dev-client/src/screens/HomeScreen.tsx (hunk 1 above)
Comment (paulschreiber) [reply dev-client/src/screens/HomeScreen.tsx:342]: I tried that, but it doesn't align properly: https://github.com/GeekyAnts/NativeBase/issues/5296

**Review by david-code — APPROVED** [review]

**Pull request merged**